Antidote/ChatViewController.m

```diff
@@ -373,4 +384,21 @@ - (void)updateLastReadDate
     [[AppContext sharedContext].profileManager.toxManager.objects changeChat:self.chat lastReadDateInterval:interval];
 }
 
+- (void)startCallButtonPressed
+{
+    OCTSubmanagerCalls *manager = [AppContext sharedContext].profileManager.toxManager.calls;
+
+    OCTCall *call = [manager callToChat:self.chat enableAudio:YES enableVideo:NO error:nil];
+
+    if (! call) {
+        return;
+    }
+
+    DialingCallViewController *dailingCallViewController = [[DialingCallViewController alloc] initWithChat:self.chat submanagerCalls:manager];
+
+    dailingCallViewController.modalInPopover = YES;
+    dailingCallViewController.modalPresentationStyle = UIModalPresentationOverFullScreen;
+    [self presentViewController:dailingCallViewController animated:YES completion:nil];
```

Comment:
Dialing controller should be put inside navigation (with hidden navigation bar). That way it would be able to push Active controller